ProcessMaker/Http/Controllers/Auth/LoginController.php

```diff
@@ -121,6 +117,28 @@ protected function getDefaultSSO(array $addons): string
         return '';
     }
 
+    protected function getLoginDefaultSSO()
+    {
+        $defaultSSO = '';
+        if (class_exists(\ProcessMaker\Package\Auth\Database\Seeds\AuthDefaultSeeder::class)) {
+            $defaultSSO = Setting::byKey(
+                \ProcessMaker\Package\Auth\Database\Seeds\AuthDefaultSeeder::SSO_DEFAULT_LOGIN
+            );
+        }
+
+        return $defaultSSO;
+    }
+
+    protected function getPmLogin()
+    {
+        $pmLogin = 'ProcessMaker';
+        if (class_exists(\ProcessMaker\Package\Auth\Database\Seeds\AuthDefaultSeeder::class)) {
+            $pmLogin = \ProcessMaker\Package\Auth\Database\Seeds\AuthDefaultSeeder::PM_LOGIN;
```

Comment:
Since this class is used multiple times, consider to declare the class in the use section
`use ProcessMaker\Package\Auth\Database\Seeds\AuthDefaultSeeder;`